src/Indexer.js

```diff
@@ -15,6 +15,10 @@ export default class Indexer {
     this.logger = new Logger()
     this.knownIndexResults = ['created', 'updated']
     this.knownDeleteResults = ['deleted']
+    this.indexers = {
+      sinopia_templates: SinopiaTemplateIndexer,
+      sinopia_resources : ResourceIndexer
```

Comment:
Stray space after `sinopia_resources` that I would have expected `eslint` to catch. No biggie.